sdk/core/core-client-rest/test/sendRequest.spec.ts

```diff
@@ -83,6 +83,35 @@ describe("sendRequest", () => {
         contentType: "multipart/form-data",
       });
     });
+
+    it("should handle request body as FormData with array of binary", async () => {
+      const expectedFormData = { fileName: "foo.txt" };
+      const mockPipeline: Pipeline = createEmptyPipeline();
+      mockPipeline.sendRequest = async (_client, request) => {
+        assert.equal(request.formData?.fileName, "foo.txt");
+        assert.isArray(request.formData?.files);
+        const files = request.formData?.files as Blob[];
+        assert.lengthOf(files, 2);
+
+        assert.instanceOf(files[0], Blob);
+        assert.instanceOf(files[1], Blob);
+        assert.sameOrderedMembers(
+          [...new Uint8Array(await (request.formData?.files as Blob[])[0].arrayBuffer())],
```

Comment:
can't we re-use our `files` reference here?
```suggestion
          [...new Uint8Array(await files[0].arrayBuffer())],
```